woocommerce/woocommerce
# Fix: Set correct prices_include_tax for manually created orders based on store tax settings

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

This PR fixes an issue where manually created orders in WooCommerce do not correctly set the `prices_include_tax` value according to the store's tax settings.

I couldn't reproduce any issue with regular WC Orders, however, this issue causes errors with checkout totals when a customer pays for a subscription renewal order for a subscription that was created via Admin.

Because a subscription is just a custom order type, I figured it would be better to fix this in core.

See https://github.com/Automattic/woocommerce-subscriptions-core/issues/682

Screenshots of the issue:

Renewal Order totals (looks correct):
![image](https://github.com/user-attachments/assets/e70453f8-9cc5-4136-a598-19750df85796)

Checkout totals when paying for the manual renewal (incorrect :x:):
![image](https://github.com/user-attachments/assets/ca42d447-4822-4bc7-84f4-e0fc441919bc)

Closes #51426.

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to **WooCommerce > Settings > Tax** and set "Prices entered with tax" to "Yes."
2. Navigate to **WooCommerce > Orders > Add Order** and create a new pending order.
3. Query the `wc_order_operational_data` table for the new Order ID.
4. Confirm that `prices_include_tax` is set to `1` (indicating prices include tax).
5. Ensure that the order totals, when processed via the Order Pay page, include the correct totals.

### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
